### ISSUE TITLE
learning rate set

### ISSUE THREAD

**Issue by He-Yunqian** [traveller59/second.pytorch#213]
Hi, thanks for your awesome code.
I would like to know how you set the learning rate.
I noticed that you trained with 165 EPOCH, did you set the learning rate the same as the upload config? Or reduce it, cause it is related to the total steps. 
Would you like to share some experience?

Thank you very much
